webapp/src/components/Question.vue

```diff
@@ -100,8 +105,19 @@ export default {
     });
   },
   methods: {
+    typeWrite(){
+            this.title = "";
+           this.titleToWrite.description.split("").forEach((letra ,  i ) => {
+            setTimeout(() => {
+                this.title+= letra;
+            }, 20 * i )
+        });
+      },
     nextQuestion() {
       this.currentQuestion = this.questionList.shift();
+      this.titleToWrite = this.currentQuestion;
```

Comment:
A gente não precisa setar esse title to write.

Reply:
Se não setarmos o titleToWrite, ele não imprimi as perguntas

Reply:
![image](https://user-images.githubusercontent.com/48883646/68678203-47437000-053c-11ea-9c42-25c5077897ff.png)

Reply:
Sim, por que temos que mudar a lógica que é feita. Minha ideia é quebrar direto o que já tem no `currentQuestion.description`